installation.md

```diff
@@ -72,6 +72,15 @@ ruby --version    # 2.7.2
 bundler --version # 2.2.0
 rails --version   # 6.1.0
 ```
-
+* We need `nodejs` package for a few of the JS gems. So you can install `nodejs` by running the following command:
+```bash
+sudo apt install nodejs
+```
+* We need `yarn` for `webpacker`which is a JavaScript pre-processor. To [install yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable) run the following commands:
```

Comment:
Yarn is a package manager and webpacker is the rails-version of webpack, a module bundler.